lib/versioning/VersionID.js

```diff
@@ -138,7 +142,7 @@ function encode(str) {
  * @param {string} str - the encoded versionId to decode
  * @return {(string|Error)} - the decoded versionId or an error
  */
-function decode(str) {
+function legacyDecode(str) {
```

Comment:
instead of calling this legacy ( which just describes the time in which it was created ) calling it like hexEncode/Decode which describes what it is doing.